convert_counts_to_vcf.py

```
@@ -311,13 +312,22 @@ def write_variants_to_vcf(
-            ):
+
+            # Skip missing
+            if value is None or value == "":
+                continue
+
+            # Skip zero, including "0", 0, 0.0, np.int64(0) etc
+            try:
+                if float(value) == 0:
+                    continue
+            except Exception:
+                pass
```

Ruff (0.14.14)

`try`-`except`-`pass` detected, consider logging the exception

(S110)